constants.go

```diff
@@ -582,11 +585,25 @@ const (
 // MinClientVersion is the minimum client version required by the server.
 var MinClientVersion string
 
+// S3AllowedACL is the set of canned ACLs that S3 accepts
+var S3AllowedACL map[string]struct{}
```

Comment:
It looks like you only need this in one package, so I would define it in the package where it is used and unexport it. 

Less things can go wrong this way 🙂